pages/about.html

```diff
@@ -3,7 +3,103 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <meta
+      name="author"
+      content="Greta Masliukova, Ricardo Vicente, Ilona Liashenko, Ielyzaveta Shabetnyk, Group2"
+    />
+    <meta name="description" content="The Pink Bear restaurant" />
+    <meta
+      name="keywords"
+      content="Restaurant, Pink Bear, food, food menu, services, website"
+    />
+    <link rel="stylesheet" href="../css/normalize.css" />
+    <link rel="stylesheet" href="../css/styles.css" />
+    <link rel="stylesheet" href="../css/about-styles.css" />
+    <title>About Us</title>
   </head>
-  <body></body>
+  <body>
+    <header>
+      <nav id="page-navigation" class="main-nav">
+        <ul>
+          <li>
+            <a href="../index.html"
+              ><img
+                src="../assets/svg/paw-solid.svg"
+                alt="paw-logo"
+                class="paw-logo"
+            /></a>
+          </li>
+          <div class="main-nav__flex">
+            <li><a href="../pages/about.html" class="f-s36">About us</a></li>
+            <li>
+              <a href="../pages/menu.html" class="f-s36">Menu</a>
+            </li>
+            <li>
+              <a href="../pages/application.html" class="f-s36">Careers</a>
+            </li>
+            <li>
+              <button class="primary-btn mr-60">
+                <a href="../pages/book-table.html" class="caps-text f-s32"
+                  >Order now</a
+                >
+              </button>
+            </li>
+          </div>
+        </ul>
+      </nav>
+    </header>
+    <main class="main-content">
+      <section class="founder-section">
+        <img
+          src="../assets/svg/original.svg"
+          alt="pink-bear"
+          class="founder-section__img"
+        />
+        <div class="founder-section__text">
+          <h1 class="founder-section__title ff-arial f-s40 fw700 lh-46">About our founder</h1>
```

Comment:
I would recommend to use <h2> here instead of <h1> as h1 tag defines the title of the whole page, not it's sections and should be used only once